packages/mu-transport/index.js

```diff
@@ -39,17 +40,47 @@ module.exports = function transport (createDriver, mu, opts) {
     tearDown: driver.tearDown
   }
 
+
+
+  function constructFailResponse (err, msg) {
+    var pkt = { err: err,
+                response: {},
+                protocol: {}}
+    var elt = muid
+
+    /* istanbul ignore else  */
```

Comment:
Agreed - but IMO the occasional pragma to maintain 100% is worth it - branch coverage is sometimes too fiddly :)